library/src/main/res/values-sw600dp/bool.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <bool name="isTablet">true</bool>
```

Comment:
Delete this file, I don't think it's needed any longer

Reply:
Done.